### PR TITLE
OCLOMRS-449: Show distinction between active and retired concepts

### PR DESCRIPTION
**Pull request by brucemakallan**
# JIRA TICKET NAME:
[Show distinction between active and retired concepts](https://issues.openmrs.org/browse/OCLOMRS-449)

# Summary:
Show retired concepts as strikethrough text

### CODE REVIEW

**Comment by dkayiwa** [comment]
Any reason why this is still open for a full 3 days?

**Comment by brucemakallan** [comment]
> Any reason why this is still open for a full 3 days?

Darius asked me to hold further action until clarification is given in the Thursday sync.

File: src/tests/dictionaryConcepts/container/DictionaryConcepts.test.jsx [hunk 1]
```diff
@@ -61,6 +62,23 @@ describe('Test suite for dictionary concepts components', () => {
...
+      org: {},
+      locationPath: {},
+      showDeleteModal: jest.fn(),
+      handleDelete: jest.fn(),
+      conceptLimit: 1,
+      closeDeleteModal: jest.fn(),
+      handleDeleteMapping: jest.fn(),
+      showDeleteMappingModal: jest.fn(),
+    };
+    const wrapper = mount(<ConceptTable {...props} />);
```
Comment (dkayiwa) [inline src/tests/dictionaryConcepts/container/DictionaryConcepts.test.jsx]: I thought the test would expect something about the different display between retired and unretired concepts.

Comment (brucemakallan) [reply src/tests/dictionaryConcepts/container/DictionaryConcepts.test.jsx]: The test checks to see if this component will render when retired concepts are present.
`concepts: [{ ...concepts, retired: true }],`

Comment (dkayiwa) [reply src/tests/dictionaryConcepts/container/DictionaryConcepts.test.jsx]: If concepts are still present, but not retired, will this test fail?

Comment (brucemakallan) [reply src/tests/dictionaryConcepts/container/DictionaryConcepts.test.jsx]: The test would pass. Let me make it more specific.

Comment (dkayiwa) [reply src/tests/dictionaryConcepts/container/DictionaryConcepts.test.jsx]: So it is not testing what it is supposed to.

Comment (brucemakallan) [reply src/tests/dictionaryConcepts/container/DictionaryConcepts.test.jsx]: Thanks for pointing it out. I have made it more specific. The test would now fail if there are no retired concepts. It looks for the exact HTML tag that creates strikethrough text.

Comment (dkayiwa) [reply src/tests/dictionaryConcepts/container/DictionaryConcepts.test.jsx]: Shouldn't this correspondingly change? 'should render the Concept Table without breaking'

Comment (brucemakallan) [reply src/tests/dictionaryConcepts/container/DictionaryConcepts.test.jsx]: Yes. I've changed it to "should contain strikethrough text for retired concepts"

**Pull request merged**